dockerfiles/Dockerfile-azkaban

```diff
@@ -0,0 +1,27 @@
+###
+# Instructions - run these commands from the directory where Dockerfile-azkaban is located
+#
+# Build command:
+# (Note - make sure at least 4G are available to your docker VM if not on Linux, Mac default is 2G)
+# (Also this takes 10-20 minutes. Would LOVE for a docker expert to advise how to improve speed :) )
+# docker build -m 4G -f Dockerfile-azkaban .
+#
+# Run command:
+# docker run -d -p 8081:8081 <YOUR BUILT CONTAINER HASH>
+#
+# Then you can view the solo-server at localhost:8081
+#
+
+FROM openjdk:8
+
+# download Azkaban from OS repo and checkout version 3.37.0
+RUN git clone https://github.com/azkaban/azkaban.git && cd azkaban && git reset --hard ce1e17b && echo "lol"
```

Comment:
To speedup, you could clone HEAD only : 

`git clone --depth=1  -b  3.37.0 url`

Then, you can remove : `git reset`

I suggest to remove  `&& echo "lol"` too. Let git "cry" then you can see if you got an error.

Reply:
Yes! Good idea. And the `&& echo "lol"` was accidental - I was using it for testing and forgot to remove it. Thanks and good catch!

Reply:
I think I'm going to avoid cloning HEAD only actually - I don't want to accidentally pick up buggy new commits. And I'm not seeing too much delay from the clone step. I think breaking it up into multi-stage will be really the way to go.

Reply:
@reallocf My bad, you will not checkout HEAD if you get the branch directly with  3.37.0. Good catch !